omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
+
+# Global FIPS override flag.
+if windows? || rhel?
+  override :fips, enabled: true
+end
 
 dependency "preparation"
-dependency "chef"
-dependency "pry"
+dependency "rb-readline" if windows?
```

Comment:
I was optimizing for build-time.  I was putting everything that could go before "chef" in front of it.